app/routes/docs.notices.schema.practices.mdx

```diff
@@ -0,0 +1,26 @@
+---
+handle:
+  breadcrumb: Schema Practices
+---
+
+# Best Practices for Schema
+
+### Recommended Unit System for Your Schema
+
+To maintain a consistent database for the same property reported by different instruments, it is best to primarily use International System of Units (SI) or standard units to ensure uniformity across all notices types. One can easily access unit conversion software, such as astropy, for internal team use.
+
+### Choosing between "allOf", "anyOf", or "oneOf"
+
+We recommend using the "allOf" option because none of the core schema properties are specified as required. However, the flexibility of not defining the required properties while using "anyOf" can lead to false positives, as technically, anything will be validated against it.
+
+### Referencing Coincident Detection with a Different Instrument
+
+We have designed a [follow-up core schema](https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/core/FollowUp.schema.json) for referencing instrument properties. Other instruments involved in follow-up studies can use this schema.
+
+### Maintaining the Schema Version
+
+The GCN Team manages schema versioning. To stay informed about new releases, follow the [GCN-Schema](https://github.com/nasa-gcn/gcn-schema) repository for notifications. Updates are categorized into three levels: major, minor, and patch, depending on the kind of update according to [Semantic Versioning](https://semver.org/) rules.
```

Comment:
@dakota002, @Courey, would you please help with making this section a bit more specific? I.e., explain that patch changes are always backwards and forward compatible, minor changes are always forwards compatible, and major changes break compatibility?

Also we need to communicate to producers that in their alerts they must refer to tagged, versioned, schema, and should _never_ refer to the main or latest stable version --- and explain why.